### PR TITLE
improve/v1-181 | Changed queryKey strings to constants

### PR DESCRIPTION
**Pull request by PechurinaLena**

### CODE REVIEW

File: frontend/src/constants/queryKeyConstants.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  myCourses: 'myCourses',
+  employeeProfile: 'EmployeeProfile',
+  courseRequests: 'courses-requests',
+  employeesLis: 'employees-list',
+  profile: 'profile',
+  clientCourseInfo: 'ClientCourseInfo',
+  paginatedClientCoursesList: 'paginatedClientCoursesList',
+  courseTest: 'CourseTest',
+  getTestTime: 'getTestTime',
+};
```
Comment (ostapdans99) [inline frontend/src/constants/queryKeyConstants.ts:14]:
```suggestion
export const QUERY_KEYS = {
    courseInfo: 'CourseInfo',
    paginatedCoursesList: 'PaginatedCoursesList',
  ....
};
```

it is better to write data in the same style.

File: frontend/src/constants/queryKeyConstants.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export const queryKeyConstants = {
```
Comment (irina-sinitsyna) [inline frontend/src/constants/queryKeyConstants.ts:1]:
```suggestion
export const QUERY_KEY = {
```

File: frontend/src/constants/queryKeyConstants.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  courseAndMaterials: 'CourseAndMaterials',
+  courseInfo: 'CourseInfo',
+  paginatedCoursesList: 'paginatedCoursesList',
+  myCourses: 'myCourses',
+  employeeProfile: 'EmployeeProfile',
+  courseRequests: 'courses-requests',
+  employeesLis: 'employees-list',
```
Comment (irina-sinitsyna) [inline frontend/src/constants/queryKeyConstants.ts:8]:
```suggestion
  employeesList: 'EmployeesList',
```

File: frontend/src/constants/queryKeyConstants.ts (hunk 1 above)
Comment (irina-sinitsyna) [reply frontend/src/constants/queryKeyConstants.ts:14]: agree with @ostapdans99

**Review by irina-sinitsyna — CHANGES_REQUESTED** [review]

File: frontend/src/constants/queryKeyConstants.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+export const QUERY_KEYS = {
+  courseAndMaterials: 'CourseAndMaterials',
+  courseInfo: 'CourseInfo',
+  paginatedCoursesList: 'PaginatedCoursesList',
+  myCourses: 'MyCourses',
+  employeeProfile: 'EmployeeProfile',
+  coursesRequests: 'CoursesRequests',
+  employeesList: 'EmployeesList',
+  profile: 'profile',
```
Comment (ostapdans99) [inline frontend/src/constants/queryKeyConstants.ts:9]:
```suggestion
  profile: 'Profile',
```
:)

**Review by ostapdans99 — APPROVED** [review]

File: frontend/src/constants/queryKeyConstants.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+export const QUERY_KEYS = {
```
Comment (SaXaPhonist) [inline frontend/src/constants/queryKeyConstants.ts:1]: make Query_keys as enum

**Review by SaXaPhonist — CHANGES_REQUESTED** [review]

**Comment by ostapdans99** [comment]
> maybe QUERY_KEYS will be better used as enum

Yep, we can use it, but in this case, we won't use this for type declarations. In general we can use enums in everywhere or in cases like this - just const object

**Pull request merged**